src/AdminCommands.ts

```diff
@@ -369,6 +369,7 @@ export class AdminCommands {
             try {
                 let matched = false;
                 this.yargs.parse(argv, {
+                    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
```

Comment:
I think the rule is not cool with us using `foo ? a() : b()` to conditionally call functions instead of assigning the return value of the entire term.